src/components/header/authorizedUser/__tests__/index.test.tsx

```diff
@@ -0,0 +1,12 @@
+import { customRender } from '__mocks__/customRender'
+import { AuthorizedUser } from '..'
+
+jest.mock('./images/logo.png', () => 'logo')
```

Comment:
why do you need to mock it?

Reply:
because it throws the error "symbol is not a function". It's because of the picture

Reply:
where is the picture used in this component?

Reply:
21 lines in header/index.tsx

Reply:
bit it is not a header component